packages/venia-ui/lib/components/Postcode/postcode.js

```diff
@@ -25,27 +19,18 @@ const Postcode = props => {
 
     const { formatMessage } = useIntl();
 
-    const hasInitialized = useRef(false);
-    const countryCodeField = 'country';
-    const countryFieldState = useFieldState(countryCodeField);
-    const { value: country } = countryFieldState;
-
-    const postcodeInputFieldApi = useFieldApi(fieldInput);
+    const fieldLabel =
+        label ||
+        formatMessage({
+            id: 'postcode.label'
```

Comment:
I think we still want defaultMessage here.

```suggestion
            id: 'postcode.label',
            defaultMessage: 'ZIP / Postal Code'
```